pkg/commands/drift/statefiles/statefiles.go

```diff
@@ -129,20 +199,54 @@ func (c *DriftStatefilesCommand) Process(ctx context.Context) error {
 		With("github_repo", c.GitHubFlags.FlagGitHubOwner)
 
 	logger.DebugContext(ctx, "starting Guardian drift statefiles")
+
+	// Clone all git repositories.
+	repositories, err := c.githubClient.ListRepositories(ctx, c.GitHubFlags.FlagGitHubOwner, &githubAPI.RepositoryListByOrgOptions{})
```

Comment:
This func is now 120 lines and getting a little hard to read even with the clear inline documentation you have. Is it possible to encapsulate some sub-logic into separate functions?